bk-api/bk_api.py

```diff
@@ -1040,18 +988,18 @@ def create_tls_cert_for_node(bee_db, node_id, beehive_obj , force=False):
 
 # /node/<node_id>
 class Node(MethodView):
+
+    # TODO(sean) We should review the public endpoints. I personally find it odd that we can post to /node
+    # but must get node info from /state.
```

Comment:
maybe have this just return the results of the `/state` end-point to keep back-wards compatibility ?

Reply:
Right. That's what I was thinking too. I know we have stuff that depends on /state, so I wouldn't want to break that.